### DISCUSSION TITLE
zh/guide/drivers/Terabox

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#112] [Announcements]
# zh/guide/drivers/Terabox

刷新令牌 点此登录 (https://www.terabox.com/)进行获取刷新令牌 Terabox 添加方式 add Terabox 注意事项：根文件路径默认是 / 如果你不想写根目录 / 要写其他目录; 挂载其他单个子目录，放一张图参考一下吧; ;

https://alist.nn.ci/zh/guide/drivers/Terabox.html

**Comment by Chengfanfu** [comment]
挂载上了，但是alist里不显示内容是怎么回事呢

**Comment by chairmansu** [comment]
掛上讀取正常
測試上傳20g以上大檔案

**Reply by chairmansu** [reply]
測試下來
沒辦法偷上傳20g以上檔案
可能要付費會員才能

**Reply by pipinian** [reply]
不开会员不是最大上传是4GB吗，你怎么能上传大文件

**Reply by chairmansu** [reply]
我有開會員
上傳目前沒辦法超出20g
其餘正常

**Reply by chairmansu** [reply]
會員部分是免費會員
並沒有付費

**Reply by imoldchen** [reply]
我也开了会员，测试10G的文件上传成功，但是不会显示出来，上传了个寂寞

**Comment by cosmosdog** [comment]
还是挂不上挂上了显示空白，貌似cookid加了很多玩意进去。按照教程复制cookid就空白

**Reply by chairmansu** [reply]
複製Cookie之前
也是一直找不到
按f5網頁重整幾次後才順利找到cookie

**Reply by zi4dbek** [reply]
为什么我能一次过？

**Reply by brysonhwang** [reply]
可以告诉我如何挂载吗？需要填什么选项，cookie也是对的，就是我挂全局也不能通过，一直显示本地区不能使用

**Comment by manesec** [comment]
有沒有人知道cookie多久過期呢？

**Comment by brysonhwang** [comment]
全局代理也挂不上不知道怎么回事？需要勾选web代理，webdav策略选哪个？有知道的望不吝赐教，谢谢

**Comment by zi4dbek** [comment]
发件人: ***@***.*** ***@***.***> 代表 brysonhwang
发送时间: 2023年2月1日 16:02
收件人: alist-org/docs ***@***.***>
抄送: zi4dbek ***@***.***>; Comment ***@***.***>
主题: Re: [alist-org/docs] zh/guide/drivers/Terabox (Discussion #112)

可能不是原生外国ip，或者被百度识别到是一个节点，建议使用国外服务器，这可能会有效。

**Comment by zi4dbek** [comment]
发件人: ***@***.*** ***@***.***> 代表 brysonhwang
发送时间: 2023年2月1日 16:00
收件人: alist-org/docs ***@***.***>
抄送: zi4dbek ***@***.***>; Comment ***@***.***>
主题: Re: [alist-org/docs] zh/guide/drivers/Terabox (Discussion #112)

 

要让alist吃到代理流量，建议挂vpn使用，并且开全局，或者使用国外服务器

**Comment by zi4dbek** [comment]
发件人: ***@***.*** ***@***.***> 代表 brysonhwang
发送时间: 2023年2月1日 16:00
收件人: alist-org/docs ***@***.***>
抄送: zi4dbek ***@***.***>; Comment ***@***.***>
主题: Re: [alist-org/docs] zh/guide/drivers/Terabox (Discussion #112)

 

Webdav策略选择默认的302，或者本地代理
（要梯子）

**Comment by brysonhwang** [comment]
谢谢回答，之前操作就是原生境外IP，而且开了全局，还是一直无法挂载，没有境外服务器看来有点难度了😅



Sent from Mail.ru app for iOS


Wednesday, February 1, 2023 at 18:51 +0800 from ***@***.***  ***@***.***>:

**Comment by Jobove** [comment]
{"error_code":31362,"error_msg":"sign error","error_info":"","request_id":8837891654503436955}

**Reply by Jobove** [reply]
这是什么问题呢

**Comment by itfanr** [comment]
Failed init storage: terabox is not yet available in this area

**Reply by zi4dbek** [reply]
terabox屏蔽了大陆ip地址，似乎梯子也不管用

**Reply by zi4dbek** [reply]
没有境外服务器的话有点难度了

**Reply by itfanr** [reply]
> 没有境外服务器的话有点难度了

感谢。那算了，1T空间不小，但是不知道会不会限速……对我没啥吸引力了

**Reply by xj9696** [reply]
这个问题我偿试过无数次，想过无数可能。最终可能找到答案。Terbox挂载时，无需任何其他设置，基本就好了。 “terabox is not yet available in this area”的出现就是很多时候默认不是走代理的，所以需要强制terabox.com走代理，我试过很多科学上网工具，目前发现ssr+强制代理也不行，但password就可以，理论原因不明，我是用笨办法一一信偿试试出来的。目前我两台openwrt挂裁alist-Tearbox均没有问题。

**Comment by pipinian** [comment]
上传1g以上的大文件都会失败，因为网页端的上传有大小限制，而电脑客户端没有这个限制，所以能不能使用客户端挂载

**Reply by simplove** [reply]
挂载好了，不过墙内无法下载。

**Reply by brysonhwang** [reply]
> 挂载好了，不过墙内无法下载。请问墙内如何挂载呢？全局也失败

**Reply by pipinian** [reply]
我不知道墙内怎么挂载，我用的海外vps

**Reply by pipinian** [reply]
可以上传大文件，一切正常

**Comment by simplove** [comment]
挂好测试了下，貌似文件大小超过50M就不给下载了

**Comment by Steven-Zhl** [comment]
## 一种可用的配置方案
### 条件
* 本机已挂系统代理且可用
### 配置
* 打开“Web代理”
* “WebDAV策略”选“本地代理”
* “下载代理URL”不填
* “下载接口”选非官方

**Comment by Jsonl95** [comment]
tls: failed to verify certificate: x509: certificate signed by unknown authority

這是什麼問題？

**Comment by xixifaha** [comment]
甲骨文vps上的alist挂载的这个，webdav 本地代理，自己电脑webdav 还是没法下载

**Comment by zhoupingxiao** [comment]
无法上传了。

**Reply by a10935177911** [reply]
JsToken--在云盘主页右键选择页面来源复制全部文字,到文件党里面,之后搜寻文字JsToken,搜寻到之后的后面%22xxxxxxxxxxxxxxxxxxx%22,复制xx文字贴上JsToken,选择360定向搭配CloudDrive2即可电脑上存

**Comment by hosonla** [comment]
請教Terabox上傳檔案錯誤顯示 [terabox] failed to precreate file, errno: %!s(int=4000023)
有網友說手动更新 JsToken 后可正常上传。
請教要如何查詢 Terabox的JsToken? 或是告知此錯誤如何解決?

**Reply by a10935177911** [reply]
JsToken--在雲盤主頁右鍵選擇頁面來源複製全部文字,到文件黨裡面,之後搜尋文字JsToken,搜尋到之後的後面%22xxxxxxxxxxxxxxxxxxx%22,複製xx文字貼上JsToken,選擇360定向搭配CloudDrive2即可電腦上存

**Comment by youkarin** [comment]
我在网页里上传的话,能显示成功上传.但刷新后还是什么都没有.导致一些软件想在里面写入数据都失败.别的网盘倒是可以

**Comment by hzj629206** [comment]
国外实测可以挂载，但内容始终为空，也不报错。选择的是Crack API.

**Reply by hzj629206** [reply]
failed to check login status according to cookie

重新编辑出现些错误，看上去 GG 了。

**Reply by qifeibro** [reply]
我是能看见文件，但是播放不了视频。

**Reply by qifeibro** [reply]
把api改成crack就好了

**Comment by pipinian** [comment]
Failed init storage: failed to check login status according to cookie

**Reply by pipinian** [reply]
不知道为什么不能挂载了

**Reply by michealhansun** [reply]
再国外VPS搭建alist挂载Terabox，国内下载还是需要挂代理才能下载，开启Web 代理 也不行

**Comment by gcien34** [comment]
Clash X Pro 开增强模式能正常使用，其他不知道怎么样

**Comment by yohototo** [comment]
百度系的就是麻烦，只能修改文件夹，上传很大概率会无效

**Reply by gesibo** [reply]
我在配置的时候无论是复制%22xxxxx%22，还是用抓包的csrf":"Zmo2XVtW-VcyvFBRJB7_EPULcO7XWQm0"，都提示的Failed init storage: failed to check login status according to cookie，用的是chrome浏览器，无论哪里环境下都无法配置成功

**Reply by orayneo1984** [reply]
Terabox 现在配置 cookie 都提示 Failed init storage: failed to check login status according to cookie
试了几个游览器都不行，是不是被和谐了？

**Comment by 816633** [comment]
上传显示：[terabox] failed to precreate file, errno: -7
